### DISCUSSION TITLE
如何新建 Command IDs ？

### DISCUSSION THREAD

**Discussion by Moli-X** [sumatrapdfreader/sumatrapdf#3353] [General]
如何新建 Command IDs ？

**Comment by GitHubRulesOK** [comment]
@Moli-X 
English please, and please give more explanation

>Command IDs 
How to create a new one ？

Command ID s such as `CmdClose` are labels for native functions, thus there is not *always* one for a function, nor a function for one
They are assigned in the application code.

For Example
`CmdCreateAnnotCircle` = Not Yet Implemented  

Some may be conditional so need to be over annotation for
`CmdDeleteAnnotation`

or no Command ID for
`CmdSaveAnnotNewFile` = use "Save As"
